### PR TITLE
Hide connector scaffolding behind a script

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
Hide connector scaffolding behind a script. Helps with https://github.com/airbytehq/airbyte/issues/725

## How
Add a new command to the `manage.sh` script. I had to rename the scaffold names so they can be passed in as argument without breaking

### CODE REVIEW

File: docs/tutorials/toy-connector.md [hunk 1]
```diff
@@ -73,10 +73,7 @@ Switched to a new branch 'sherifnada/source-connector-tutorial'
 Airbyte provides a code generator which bootstraps the scaffolding for our connector. Let's use it by running:
 
 ```bash
-$ cd airbyte-integrations/connector-templates/generator
-# Install NPM from https://www.npmjs.com/get-npm if you don't have it
-$ npm install
-$ npm run generate
+$ ./tools/integrations/manage.sh scaffold
```
Comment (cgardens) [inline docs/tutorials/toy-connector.md:76]: i don't think this is a DX friendly choice. the `manage.sh` script is intimidating and filled with all sorts of deploy / publishing logic. wrapping 2 very common place commands that every developer has used before inside a really huge script doesn't seem great from the DX point of view. trying to put myself in the shoes of a contributor who is coming to our project for the first time, i would probably not be willing to debug an error in `manage.sh`, but if either `npm install` or `npm run generate` failed, i would at least try to debug it.

i would suggest:
1. not wrapping these
2. if you do think you need a script, put it in a separate script that is just for this case and not full of other stuff.

Comment (sherifnada) [reply docs/tutorials/toy-connector.md:76]: is a contributor expected to use this script? If yes then I agree but if not and this is for CI purposes then I'm fine with it

Comment (cgardens) [reply docs/tutorials/toy-connector.md:76]: my interpretation was that it was used by the CI and a contributor. If I am misunderstanding and the contributor does _not_ need to interact with this then I am ambivalent.

Comment (michel-tricot) [reply docs/tutorials/toy-connector.md:76]: one of the reason I had in mind when hiding it in a script was to not duplicate the generation commands in many places (docs, gradle, tutorials...). 

I'll just revert the contributor piece but I will keep the command in the script so it is easier to invoke from gradle.

Comment (michel-tricot) [reply docs/tutorials/toy-connector.md:76]: tried to inspire myself from rails where you have one super command that can do a lot of things, and everything is discoverable through that command

**Review by sherifnada — APPROVED** [review]

**Pull request merged**